config/config/sc-config.yaml

```diff
@@ -998,11 +998,11 @@ fluentd:
       enabled: true
       ephemeralVolumes:
         enabled: false
-      schedule: 0 22 * * *
+      schedule: 0 21/6 * * *
```

Comment:
will this run every 6th hour between 21 to 23?

Reply:
Yeah, I will change it as it didn't work as I expected.

Reply:
Now it's changed, also moved the window for when it does it so it is well before the default sync schedule, and set so the retention runs one hour after the last run each day.